server/services/datastoreSearchService.test.ts

```diff
@@ -242,7 +241,57 @@ describe('Datastore Search Service', () => {
         data: {},
       }
 
-      expect(datastoreSearchService.search(input)).rejects.toThrow('Error retrieving search results')
+      expect(datastoreSearchService.submitSearchQuery(input)).rejects.toThrow('Error submitting search query')
+    })
+  })
+
+  describe('getSearchResults', () => {
+    it('submits a request containing a query execution ID and returns search results', async () => {
+      jest.spyOn(datastoreClient, 'getSearchResults').mockResolvedValue(orders)
+
+      const request = {
+        token,
+        queryExecutionId,
+      }
+
+      const result = await datastoreSearchService.getSearchResults(request)
+
+      expect(datastoreClient.getSearchResults).toHaveBeenCalledWith(request)
+      expect(result).toEqual(orders)
+    })
+
+    describe('error handling', () => {
+      // TODO: Create a working test for invalid execution ID routing.
```

Comment:
I had trouble getting tests configured correctly for this piece of error handling.

Reply:
This test now works.